feature/experimental/gribi/ate_tests/route_addition_during_failover_test/route_addition_during_failover_test.go

```diff
@@ -0,0 +1,667 @@
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package route_addition_during_failover_test
+
+import (
+	"context"
+	"encoding/binary"
+	"fmt"
+	"net"
+	"regexp"
+	"testing"
+	"time"
+
+	"github.com/openconfig/featureprofiles/internal/attrs"
+	cmp "github.com/openconfig/featureprofiles/internal/components"
+	"github.com/openconfig/featureprofiles/internal/deviations"
+	"github.com/openconfig/featureprofiles/internal/fptest"
+	"github.com/openconfig/featureprofiles/internal/gribi"
+	"github.com/openconfig/gribigo/chk"
+	"github.com/openconfig/gribigo/constants"
+	"github.com/openconfig/gribigo/fluent"
+	"github.com/openconfig/ondatra"
+	"github.com/openconfig/ondatra/gnmi"
+	"github.com/openconfig/ondatra/gnmi/oc"
+	"github.com/openconfig/ondatra/raw"
+	"github.com/openconfig/testt"
+	"github.com/openconfig/ygot/ygot"
+
+	fpb "github.com/openconfig/gnoi/file"
+	spb "github.com/openconfig/gnoi/system"
+	aftspb "github.com/openconfig/gribi/v1/proto/service"
+)
+
+func TestMain(m *testing.M) {
+	fptest.RunTests(m)
+}
+
+// Settings for configuring the baseline testbed with the test topology.
+//
+// The testbed consists of ate:port1 -> dut:port1
+// and dut:port2 -> ate:port2.
+// There are 64 SubInterfaces between dut:port2
+// and ate:port2
+//
+//   - ate:port1 -> dut:port1 subnet 192.0.2.0/30
+//   - ate:port2 -> dut:port2 64 Sub interfaces:
+//   - ate:port2.0 -> dut:port2.0 VLAN-ID: 0 subnet 198.51.100.0/30
+//   - ate:port2.1 -> dut:port2.1 VLAN-ID: 1 subnet 198.51.100.4/30
+//   - ate:port2.2 -> dut:port2.2 VLAN-ID: 2 subnet 198.51.100.8/30
+//   - ate:port2.i -> dut:port2.i VLAN-ID i subnet 198.51.100.(4*i)/30
+//   - ate:port2.63 -> dut:port2.63 VLAN-ID 63 subnet 198.51.100.252/30
+const (
+	ipv4PrefixLen       = 30
+	nhgID               = 1
+	controllerCardType  = oc.PlatformTypes_OPENCONFIG_HARDWARE_COMPONENT_CONTROLLER_CARD
+	primaryController   = oc.Platform_ComponentRedundantRole_PRIMARY
+	secondaryController = oc.Platform_ComponentRedundantRole_SECONDARY
+	switchTrigger       = oc.PlatformTypes_ComponentRedundantRoleSwitchoverReasonTrigger_SYSTEM_INITIATED
+	maxSwitchoverTime   = 900
+	configNhg           = true
+	switchover          = true
+)
+
+type flowArgs struct {
+	ipBlock          string
+	flowStartAddress string
+	flowEndAddress   string
+	flowCount        uint32
+}
+
+var (
+	dutPort1 = attrs.Attributes{
+		Desc:    "dutPort1",
+		IPv4:    "192.0.2.1",
+		IPv4Len: ipv4PrefixLen,
+	}
+	atePort1 = attrs.Attributes{
+		Name:    "atePort1",
+		IPv4:    "192.0.2.2",
+		IPv4Len: ipv4PrefixLen,
+	}
+	vendorCoreFilePath = map[ondatra.Vendor]string{
+		ondatra.JUNIPER: "/var/core/",
+		ondatra.CISCO:   "/misc/disk1/",
+	}
+	vendorCoreFileNamePattern = map[ondatra.Vendor]*regexp.Regexp{
+		ondatra.JUNIPER: regexp.MustCompile("rpd.*core*"),
```

Comment:
This looks like a daemon-specific coredump check. Isn't the objective to check for corefiles from all daemons?

Reply:
@LimeHat I have updated core file related changes.

Reply:
If this is supposed to be gribi-specific, could you please

- update the README to indicate that (currently it does not mention a specific type of coredumps, e.g. `Check for coredumps in the DUT and validate that none are present post failover`)
- update nokia pattern to `"coredump-sr_gribi_server-.*"`

thanks